src/templates/customer/write_review/not_found.template.html

```diff
@@ -1,17 +1,12 @@
-<script type="text/javascript">
-function onload(){
```

Comment:
Why are you removing this? It is an onload function to go back to the product page.

Reply:
@philcon93 the function is never executed

Reply:
So if it doesn't work we can try to understand what it was trying to do; redirect the user back to the product page. So we should use a jQuery onready/onload function to call `product_redirect ` instead of removing it.